csfieldguide/static/interactives/pixel-viewer/js/pixel-viewer.js

```diff
@@ -151,7 +151,7 @@ function setUpMode(){
 
```

Comment:
See if you can put the menu on the right hand side for large screens, so if the image is zoomed out you can at least see most of it with the menu open

Reply:
Awesome idea!